lib/thumbs/pull_request_worker.rb

```diff
@@ -1,47 +1,66 @@
 module Thumbs
   class PullRequestWorker
     attr_reader :build_dir
-    attr_reader :build_status
-    attr_accessor :build_steps
-    attr_reader :minimum_reviewers
     attr_reader :repo
     attr_reader :pr
-    attr_accessor :thumb_config
     attr_reader :log
     attr_reader :org
-    attr_accessor :client
-    attr_reader :pull_request_id
+    attr_reader :thumbs_config_default
 
     def initialize(options)
-      @repo = options[:repo]
-      @org = repo.split('/').first
-      @client = Octokit::Client.new(:netrc => true)
-      @pr = @client.pull_request(options[:repo], options[:pr])
-      @pull_request_id=get_pull_request_id
-      @build_dir=options[:build_dir] || "/tmp/thumbs/#{build_guid}"
-      persisted_build_status = read_build_status
-      @build_status = persisted_build_status || {:steps => {}}
-      @build_steps = []
+      if options[:url]
```

Comment:
The existing `repo` and `pr` options are good, but simply pasting in a `url` is easier :) . The parse was implemented super simply, supporting only https GH url for a PR.